site-manage/datemanager/impl/src/main/java/org/sakaiproject/datemanager/impl/DateManagerServiceImpl.java

```diff
@@ -1398,7 +1399,8 @@ public DateManagerValidation validateForums(String siteId, JSONArray forums) thr
 
                                 String entityType = (String)jsonForum.get(DateManagerConstants.JSON_EXTRAINFO_PARAM_NAME);
```

Comment:
this is an internationalized value and using to compare against a constant is probably not ideal.